email/emailer.go

```diff
@@ -1,56 +1,84 @@
 package email
 
 import (
-	"net"
-	"net/smtp"
-	"sync"
-
-	"github.com/HomesNZ/go-common/env"
 	"github.com/Sirupsen/logrus"
-	"github.com/jordan-wright/email"
+	"github.com/aws/aws-sdk-go/aws"
+	"github.com/aws/aws-sdk-go/aws/awserr"
+	"github.com/aws/aws-sdk-go/aws/credentials"
+	"github.com/aws/aws-sdk-go/aws/session"
+	"github.com/aws/aws-sdk-go/service/ses"
 )
 
-var (
-	contextLogger = logrus.WithField("email", "sender")
-	once          sync.Once
-	Emailer       EmailSender
-)
+// CharSet is a default AWS character set
+const CharSet = "UTF-8"
 
-type EmailSender struct {
-	Username       string
-	Password       string
-	ServerHostPort string
-}
+var awsSession *session.Session
 
-func InitEmailer() {
-	once.Do(func() {
-		Emailer = EmailSender{
-			ServerHostPort: env.MustGetString("SMTP_HOST"),
-			Username:       env.MustGetString("SMTP_USER"),
-			Password:       env.MustGetString("SMTP_PASSWORD"),
-		}
+// Init creates a new session for SES requests
+func Init() error {
```

Comment:
Since this change isn't backwards compatible anyway, why are we using an old style global with init method?